metadata.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import os
+
+# Path of .csv file
+input_path = input("Enter path of .csv metadata file: ")
+output_dir = input("Enter directory for output (leave blank for current directory): ")
+
+# Exit if path doesn't exist
+if not os.path.exists(input_path):
+    print("Invalid input path, exiting...")
+    exit()
+
+# Check if valid output dir
+if not output_dir: #if empty string
+    output_path = os.path.join('.', 'metadata.h5')
+elif os.path.isdir(output_dir): #existing directory
+    output_path = os.path.join(output_dir, 'metadata.h5')
+else: #non existant directory
+    print("Invalid output directory, exiting...")
+    exit()
+
+# Read .csv
+try:
+    metadata = pd.read_csv(input_path, sep=';')
+except:
```

Comment:
If you use `except`, you will catch all possible exceptions, which may not be related to `Incorrect format`.  In this case, it might be better to just let `pd.read_csv` fail gracefully and display the error.